cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -1879,19 +1879,26 @@ where
             .iter()
             .map(|param| param.value_type)
             .zip(builder.block_params(block).iter().copied())
-            .collect();
+            .map(|(ty, param)| Ok((ty, param, self.u.arbitrary()?)))
+            .collect::<Result<Vec<_>>>()?;
 
         // Create a pool of vars that are going to be used in this function
         for _ in 0..self.param(&self.config.vars_per_function)? {
             let ty = self.u._type((&*self.isa).supports_simd())?;
             let value = self.generate_const(builder, ty)?;
-            vars.push((ty, value));
+            let uses_stack_map = self.u.arbitrary()?;
```

Comment:
```suggestion
            let uses_stack_map = ty.bytes() <= 16 && self.u.arbitrary()?;
```

Reply:
Not sure if `cranelift-fuzzgen` will generate types larger than 16 bytes in the first place, but even if it doesn't, this is a good guard rail to have for future proofing.

Reply:
Not currently, but probably good to add this in!